python/pipeline/meso.py

```diff
@@ -1183,7 +1186,7 @@ def make(self, key):
         field_id = key['field'] - 1
         channel = key['channel'] - 1
         scan_filename = (experiment.Scan() & key).local_filenames_as_wildcard
-        scan = scanreader.read_scan(scan_filename)
+        scan = scanreader.read_scan('/data/odor_meso/' + scan_filename)
```

Comment:
use environment variable or dj.config location variable if possible, to avoid hardcode the path.